tasks/main.yml

```diff
@@ -63,7 +63,7 @@
       package:
         name: libselinux-python
         state: present
-      when: ansible_os_family == "RedHat" and ansible_distribution_major_version >= 7
+      when: ansible_os_family == "RedHat" and ansible_distribution_major_version >= "7"
```

Comment:
Looks good. I don't understand why ansible 2.7 have worked in our testing, where it's a "7" with 2.7 and earlier versions. If we run into issue in the future we could use a filter to force the output to be a string or an int.
```
      when: ansible_os_family == "RedHat" and ansible_distribution_major_version|string >= "7"
```

https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/facts/system/distribution.py